seminar6/task1.py

```diff
@@ -0,0 +1,62 @@
+#Калькулятор арифметических выражений
+
+def calc(expression, index = [0]):
+#Используется список для создания единого для всех вызовов функции счетчика текущей позиции
```

Comment:
Ваше значение по умолчанию является изменяемым. Лучше делать так:
```suggestion
def calc(expression, index=None):
#Используется список для создания единого для всех вызовов функции счетчика текущей позиции
    if index is None:
        index = [0]
```